packages/x-charts/src/ChartsLabel/ChartsLabelMark.tsx

```diff
@@ -69,14 +74,15 @@ const Root = styled('div', {
  * Generates the label mark for the tooltip and legend.
  * @ignore - internal component.
  */
-const ChartsLabelMark = consumeThemeProps(
+const ChartsLabelMark = consumeSlots(
   'MuiChartsLabelMark',
+  'legendLabelMark',
   {
     defaultProps: { type: 'square' },
     classesResolver: useUtilityClasses,
   },
   function ChartsLabelMark(props: ChartsLabelMarkProps, ref: React.Ref<HTMLDivElement>) {
-    const { type, color, className, classes, ...other } = props;
+    const { type, color, className, classes, seriesId: unused, ...other } = props;
```

Comment:
Not sure what's the best way to prevent `seriesId` from being passed to `ChartsLabelMark`. We want to pass it in case the user wants to create a slot that changes depending on the ID of the series, but it isn't used for the default component.

Reply:
you should be able to just have `seriesId` there, without the name change

Reply:
You mean without renaming the prop to `unused`? Yeah, that works as well.